src/components/StatusBarStyled.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { StatusBar } from 'react-native'
+
+import { COLORS } from '../theme/colors'
+
+export const StatusBarStyled = (props) => {
+  return (
+    <StatusBar
+      backgroundColor={props.light ? COLORS.paper : COLORS.text}
```

Comment:
i pushed a commit with that change, so you can see.

Reply:
I found it logical to call it light, when the bar itself is light. Maybe we cann all it `bg-light`? But that is still confusing, because that could be the other screen again. I don't care that much.
Did you also make it work for iphones? @khamui

Reply:
@fuzzylogic2000 but it is not only the naming, it also changes how it behaves. so if the page is white, the status bar background should be dark or should be white as well? For me it is logical to have it seamless, but I might misunderstood the issue.

For iPhones it seems to be readable:

<img width="535" alt="Bildschirm­foto 2023-03-20 um 14 27 00" src="https://user-images.githubusercontent.com/52801593/226364044-07249b62-3665-4352-8026-7f73ffcc2e3b.png">

<img width="496" alt="Bildschirm­foto 2023-03-20 um 14 26 54" src="https://user-images.githubusercontent.com/52801593/226364030-03eac6b5-6739-4a41-88d5-e0e9eaee09cf.png">

Reply:
@khamui You were right! I will squash the commits and try to only make the icons white, not the status bar black on images.